oxidation/crates/platform-async/src/platform.rs

```diff
@@ -0,0 +1,8 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
```

Comment:
we use `<module_name>/mod.rs` style in the codebase

Reply:
I've moved the file.

I've looked if we can configure clippy for the whole project:

It's not possible at the moment but https://github.com/rust-lang/cargo/issues/5034